pkg/config/env/environment_detection.go

```diff
@@ -74,28 +75,35 @@ func IsFeaturePresent(feature Feature) bool {
 }
 
 // IsAutoconfigEnabled returns if autoconfig from environment is activated or not
-func IsAutoconfigEnabled() bool {
+func IsAutoconfigEnabled(cfg model.Reader) bool {
 	// Usage of pure environment variables should be deprecated
 	for _, envVar := range []string{autoconfEnvironmentVariable, autoconfEnvironmentVariableWithTypo} {
 		if autoconfStr, found := os.LookupEnv(envVar); found {
 			activateAutoconfFromEnv, err := strconv.ParseBool(autoconfStr)
 			if err != nil {
-				log.Errorf("Unable to parse Autoconf value: '%s', err: %v - autoconfig from environment will be deactivated", autoconfStr, err)
+				log.Errorf(
```

Comment:
nit: seems like auto-formatting from IDE or something, please avoid it (and multi-line is IMHO way worse to read in these cases)